src/Driver/driver_configs.jl

```diff
@@ -17,13 +17,27 @@ struct AtmosLESSpecificInfo <: ConfigSpecificInfo end
 struct AtmosGCMSpecificInfo{FT} <: ConfigSpecificInfo
     domain_height::FT
     nelem_vert::Int
-    nelem_horz::Int
+    nelem_horiz::Int
 end
 struct OceanBoxGCMSpecificInfo <: ConfigSpecificInfo end
 struct SingleStackSpecificInfo <: ConfigSpecificInfo end
 
 include("SolverTypes/SolverTypes.jl")
 
+"""
+    ArgParse.parse_item
+
+Parses custom command line option for tuples of two integers.
+"""
+function ArgParse.parse_item(::Type{NTuple{2, Int}}, s::AbstractString)
+
+    str_array = split(s, ",")
```

Comment:
you can lstrip and rstrip the resulting split string to get rid of the whitespace issue

Reply:
I think a space after the comma will make `ArgParse` treat the second part of the tuple as a separate argument?

Reply:
I see, this is why having non-magical cli argument syntax is nice!

Reply:
I don't disagree. 😜 

Might not be the best supporting argument in favor, but gcc has command line arguments like this, in which commas cannot be followed by spaces (e.g. `-Wa,`).